### DISCUSSION TITLE
NSAdvertisingAttributionReportEndpoint IOS Setting

### DISCUSSION THREAD

**Discussion by a9848840** [airsdk/Adobe-Runtime-Support#1901] [Q&A] [answered]
I have a problem with setting NSAdvertisingAttributionReportEndpoint ,

iOS 15+ ad configuration

Info.plist, in FlashDevelop mean -->

```
<InfoAdditions>
  <![CDATA[
     <key>NSAdvertisingAttributionReportEndpoint</key>
       <string>https://appsflyer-skadnetwork.com/</string>
   ]]>
</InfoAdditions>
```

Is this correct?

Reference URL:
https://support.appsflyer.com/hc/en-us/articles/4402320969617?utm_source=hq1&amp;utm_medium=referral#configure-your-app-to-send-postback-copies-to-appsflyer

**Comment by ajwfrost** [comment, marked as answer]
That looks correct, it should inject that key and string value into the Info.plist for the generated IPA file.

If you've got the IPA file, you can unzip this and check under Payload/appname.app/Info.plist to check that the value has been correctly inserted..

thanks

**Reply by a9848840** [reply]
thank you very much

unzip my IPA
Payload/appname.app/Info.plist
get

>          <key>NSAdvertisingAttributionReportEndpoint</key>
>          <string>https://appsflyer-skadnetwor.com/</string>

:)

**Reply by shaymolho** [reply]
you have a typo  in
    <string>https://appsflyer-skadnetwor.com/</string>
it should be
     <string>https://appsflyer-skadnetwork.com/</string>
